modules/machine-health-checks-about.adoc

```diff
@@ -29,8 +29,55 @@ Consider the timeouts carefully, accounting for workloads and requirements.
 
 To stop the check, remove the resource.
 
-For example, you should stop the check during the upgrade process because the nodes in the cluster might become temporarily unavailable. The `MachineHealthCheck` might identify such nodes as unhealthy and reboot them. To avoid rebooting such nodes, remove any `MachineHealthCheck` resource that you have deployed before updating the cluster.
-However, a `MachineHealthCheck` resource that is deployed by default (such as `machine-api-termination-handler`) cannot be removed and will be recreated.
+
+[id="machine-health-checks-pause_{context}"]
+== Pausing machine health checks
+During the upgrade process, nodes in the cluster might become temporarily unavailable. The `MachineHealthCheck` might identify such nodes as unhealthy and reboot them. To avoid rebooting such nodes, pause all the `MachineHealthCheck` resources before updating the cluster.
+
+To pause the check, add `cluster.x-k8s.io/paused=""` annotation to the `MachineHealthCheck` resource. To do so, run the following command:
+
+[source,bash]
```

Comment:
Always introduce commands with `[source,terminal]` not `[source,bash]`. https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

```suggestion
[source,terminal]
```

Reply:
Accepted and updated